SwiftyStoreKit/PaymentQueueController.swift

```diff
@@ -91,6 +91,25 @@ class PaymentQueueController: NSObject, SKPaymentTransactionObserver {
         paymentQueue.remove(self)
     }
 
+	var unhandledTransactions: [Purchase] {
+		guard let completeTransactions = completeTransactionsController.completeTransactions else {
+			return []
+		}
```

Comment:
I think a `print` statement reminding the user to call `completeTransactions()` here would help:
```swift
guard let completeTransactions = completeTransactionsController.completeTransactions else {
    print("SwiftyStoreKit.completeTransactions() should be called once when the app launches. Returning no results")
    return []
}
```
Would you be able to update accordingly?